release-notes/aws/v10.1.1.md

```diff
@@ -0,0 +1,41 @@
+# :zap: Giant Swarm Release 10.1.1 for AWS is now active for you! :zap:
+
+This Node Pools release fixes a problem with CoreDNS in clusters using custom
+cluster IP ranges. Additionally CPU limits have been removed from multiple
+default apps.
```

Comment:
```suggestion
default apps to increase general reliability of these.
```

Reply:
Could add a link to why that is the case. WDYT?

Reply:
Yes good idea. @sslavic Do you have a link for this?